projects/igniteui-angular/src/lib/grids/grid.common.ts

```diff
@@ -386,6 +398,12 @@ export class IgxColumnMovingDragDirective extends IgxDragDirective {
             this.top = this._dragStartY = pageY - ((this._dragGhost.getBoundingClientRect().height / 3) * 2) - hostElemTop;
         }
     }
+
+    private _unsubscribe() {
+        if (this.subscription$) {
```

Comment:
In order this if clause to work properly you should set `this.subscription$` to null after the `this.subscription$.unsubscribe();`.